content/md/what-is/what-is-a-rule_fr.md

```diff
@@ -0,0 +1,61 @@
+---
+id: what-is-a-rule
+themes: boost-your-productivity, akeneo-concepts
+title: Qu'est-ce qu'une **règle**?
+popular: false
+ee-only: true
+related: manage-your-rules
+---
+
+# Définition d'une règle
+
+Une règle est une liste d'actions et de conditions qui permet d'automatiser l'enrichissement de vos données produit. 
+Par exemple, l'utilisation des règles permet:
+- de catégoriser automatiquement des nouveaux produits en fonction de leur famille
+- de copier la valeur d'un attribut texte dans un autre
+- de définir une valeur par défaut pour un champ vide en particulier
```

Comment:
"set a default value to an empty field"
- définir une valeur par défaut dans un champ vide